generators/client/templates/gulpfile.js

```diff
@@ -30,6 +30,8 @@ var gulp = require('gulp'),
     serve = require('./gulp/serve'),
     util = require('./gulp/utils'),
     gulpIf = require('gulp-if'),
+    inject = require('gulp-inject'),
+    angularFilesort = require('gulp-angular-filesort'),
```

Comment:
I dont think `gulp-angular-filesort` is required for our use case coz we use the same module name for all files and since all the files needs to be loaded for app to work. angular does all the injections at run time so the file order actually doesn't matter